### PR TITLE
Bugfix/SC-8851 fix broken queries in admin tables

### PR DESCRIPTION
**Pull request by Tratori**
Base links to copy
- https://ticketsystem.schul-cloud.org/browse/SC-8851

### CODE REVIEW

File: src/pages/administration/teachers/index.vue [hunk 1]
```diff
@@ -483,15 +486,20 @@ export default {
 			});
 		},
 		barSearch: function (searchText) {
+			if (this.$options.debounce) {
```
Comment (pkleybolte) [inline src/pages/administration/teachers/index.vue:489]: Why `this.$options` instead of a normal field in `data`

**Review by pkleybolte — CHANGES_REQUESTED** [review]
Tests are missing

**Comment by virgilchiriac** [comment]
@j-jmnz should this be rejected & closed?

**Comment by j-jmnz** [comment]
This PR is getting closed. This issue was solved in https://ticketsystem.hpi-schul-cloud.org/browse/SC-8880.

**Pull request closed without merging**